core/GlobalState.h

```diff
@@ -234,7 +249,11 @@ class GlobalState final {
     u2 stringsLastPageUsed = STRINGS_PAGE_SIZE + 1;
     std::vector<Name> names;
     UnorderedMap<std::string, FileRef> fileRefByPath;
-    std::vector<Symbol> symbols;
+    std::vector<Symbol> classAndModules;
```

Comment:
We expect to followup with PR that changes these to specialized types? right?

Reply:
Correct.